capture/capture.py

```diff
@@ -0,0 +1,10 @@
+from abc import ABCMeta, abstractmethod
+
+import capture
+
+class ImgCapture(metaclass=ABCMeta):
```

Comment:
I am really not sure about the use of `ABCMeta` and `@abstractmethod` here.
It seems to me that the intent here is really to declare an image capture 'interface', requiring the implementation of the `capture` method, and using a metaclass + decorator for that is overkill.
I would prefer it this way:
```
class BaseImageCaptureDevice(object):
    def capture(self, output_file):
       raise NotImplementedError
```
It achieves nearly the same behaviour (if we use a derived class that does not implement the `capture` method, an error will be raised upon calling `capture`, as opposed to when instantiating the class) - and is **much** simpler conceptually speaking.

Use cases for `ABCMeta` that I am aware of are
* tweaking the behaviour of `isinstance` and `issubclass` without actually changing the class hierarchy (for instance if you want a base Python class to be viewed as a child class of one of your own classes)
* defining subclass hooks so that a class implementing some method is automagically viewed as a child class of some other class ; for instance if your class `MyClass` implements `__contains__`,  then regardless of its class hierarchy, `issubclass(MyClass, collections.Container)` will return true.